src/components/pages/Login/index.tsx

```diff
@@ -80,8 +84,12 @@ const Login = () => {
         // @ts-ignore
         const newValue = event?.target?.value ?? "";
 
+        if(!isLoading){
```

Comment:
```suggestion
        if(isLoading) return;
```
You can stop execution of the rest of this function by returning before the code below. It's kinda cleaner.